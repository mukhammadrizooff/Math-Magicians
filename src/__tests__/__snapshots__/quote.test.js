import React from 'react';
import renderer from 'react-test-renderer';

import Quote from '../../components/quote';

it('renders correctly when there are no items', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
