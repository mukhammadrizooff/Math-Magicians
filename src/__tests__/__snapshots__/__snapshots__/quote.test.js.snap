// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="container-quote"
>
  <p>
    “Live as if you were to die tomorrow. Learn as if you were to live forever.”
  </p>
  <span>
    – Mahatma Gandhi
  </span>
</div>
`;
